demo/x-local-translate.html

```diff
@@ -18,9 +18,9 @@
     <style include="common-styles"></style>
 
     <div class="lang">
-      <span title="english">🇬🇧</span>
+      <span title="english">🇬🇧 (english)</span>
```

Comment:
Can we just have `🇬🇧  EN - FR 🇫🇷`, please? I think it looks a little bit more cleaner/symmetric.